### PR TITLE
Update logo, make clickable

### PR DESCRIPTION
**Pull request by Caseysch**
I think. I'm not sure how this TreeComposer works and these changes aren't tested.

### CODE REVIEW

**Comment by Caseysch** [comment]
Change confirmed by running locally. Before vs after for image comparison:
image

File: Source/SmallBasic.Editor/Components/Layout/MainLayout.scss [hunk 1]
```diff
@@ -26,7 +26,7 @@ main-layout {
             align-items: center;
 
             logo {
-                @include set-background-image("Logo.png", 224px, 80px);
+                @include set-background-image("logo.svg", 224px, 80px);
```
Comment (OmarTawfik) [inline Source/SmallBasic.Editor/Components/Layout/MainLayout.scss:29]: Thanks for looking into this!
Is the old logo used anywhere else? can we delete it now?

**Review by OmarTawfik — CHANGES_REQUESTED** [review]

**Comment by Caseysch2** [comment]
I don't think it's used anywhere else. It should be deleted regardless :) will delete it today

File: Source/SmallBasic.Editor/Components/Layout/MainLayout.scss (hunk 1 above)
Comment (Caseysch) [reply Source/SmallBasic.Editor/Components/Layout/MainLayout.scss:29]: Doesn't look like it's used elsewhere. Deleted!

**Pull request merged**